common/components/componentsBySection/VAR/stories/VARQ1.stories.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import VARQ1 from '../VARQ1'; 
+
+
+export default {
+  title: 'VAR/VARQ1',
+  component: VARQ1,
+};
+
+const Template = (args) => <VARQ1 {...args} />;
+
+export const Default = Template.bind({});
```

Comment:
Template.bind is  not needed any more. It's an older pattern.  The tests can be done this ways:
```
export const Default={
   args: { 
                initialValue: 0.0,
                hasError: false,
              },
   parameters: {...},
}
```